webmails/rainloop/config/nginx-rainloop.conf

```diff
@@ -0,0 +1,38 @@
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+
+    root /var/www/rainloop;
+    
+    # /dev/stdout (Default), <path>, off
+    access_log off;
+
+    # /dev/stderr (Default), <path>, debug, info, notice, warn, error, crit, alert, emerg
+    error_log /dev/stderr error;
+
+    index index.php;
+
+    location / {
+        try_files $uri $uri/ /index.php?$query_string;
+    }
+
```

Comment:
location ~* \.(?:css|js)$ {
      expires 7d;
      add_header Pragma public;
      add_header Cache-Control "public";
    }

    location ~* \.(?:gif|jpe?g|png|ico|otf|eot|svg|ttf|woff|woff2)$ {
      expires 30d;
      add_header Pragma public;
      add_header Cache-Control "public";
    }

Reply:
The Rainloop installation does not mention this: https://www.rainloop.net/docs/installation/

Do you have a use case where this is required? Then I'll add it.

Reply:
It's not required it's just to enhance performance (caching headers limit round-trips)

Reply:
Ok, I'll add it.

Reply:
Have we given up on any form of caching? The cache could be even more aggressive than suggested above as there is the version in the path for static assets.